store/sqlstore/channel_store.go

```diff
@@ -3627,7 +3638,7 @@ func (s SqlChannelStore) updateSidebarCategoryOrderT(transaction *gorp.Transacti
 		runningOrder += model.MinimalSidebarSortDistance
 	}
 
-	if _, err := s.GetMaster().UpdateColumns(func(col *gorp.ColumnMap) bool {
+	if _, err := transaction.UpdateColumns(func(col *gorp.ColumnMap) bool {
```

Comment:
Same here, why not build your SQL query by hand?

Reply:
In the followup, this one is staying as-is. `UpdateColumns` loops through each item passed in end executes an UPDATE query for each one, and I'm thinking it might have something to make that looping more efficient than writing it by hand